src/pages/hotel/search/components/room-availability/rooms/RoomList.tsx

```diff
@@ -33,6 +33,21 @@ function RoomList({
   }, [roomAvailabilityResponse])
 
   const [select, setSelect] = useState(-1)
+  const [error, setError] = useState(false)
+  const errorSection = useRef<HTMLDivElement>(null)
+
+  const scrollTo = (types: string) => {
+    let topSection
+    if (types == 'Error') {
```

Comment:
You can remove these unnecessary if checks and directly access errorSection ref.
`const scrollTo = () => {
     window.scrollTo({
        top: errorSection.current!.offsetTop,
        behavior: 'smooth',
      })
  }`